kimhaejung/services/userService.js

```diff
@@ -0,0 +1,48 @@
+const userDao = require("../models/userDao");
+
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+
+const signUp = async (name, email, password, profileImage) => {
+  const saltRounds = 10;
+  const makeHash = async (password, saltRounds) => {
+    return await bcrypt.hash(password, saltRounds);
+  };
```

Comment:
makeHash를 굳이 안 만들어도 될 것 같습니다. 함수로 만들지 말고 바로 적용해 보세요.
```suggestion
  const makeHash = async (password, saltRounds) => {
    return await bcrypt.hash(password, saltRounds);
  };
```